sdk/communication/azure-communication-phonenumbers/src/samples/java/com/azure/communication/phonenumbers/FindAndPurchasePhoneNumberExample.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.phonenumbers;
+
+import com.azure.communication.phonenumbers.models.PhoneNumberCapabilities;
+import com.azure.communication.phonenumbers.models.PhoneNumberCapabilityType;
+import com.azure.communication.phonenumbers.models.PhoneNumberSearchOptions;
+import com.azure.communication.phonenumbers.models.PhoneNumberSearchResult;
+import com.azure.communication.phonenumbers.models.PhoneNumberType;
+import com.azure.communication.phonenumbers.models.PhoneNumberAssignmentType;
+import com.azure.communication.phonenumbers.models.PhoneNumberOperation;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.PollResponse;
+
+public class FindAndPurchasePhoneNumberExample {
+    public static void main(String[] args) {
+        String connectionString = System.getenv("COMMUNICATION_CONNECTION_STRING");
+        String areaCode = System.getenv("AZURE_COMMUNICATION_AREA_CODE");
+
+        PhoneNumbersClient phoneNumberClient = new PhoneNumbersClientBuilder()
+            .connectionString(connectionString)
+            .buildClient();
+
+        PhoneNumberCapabilities capabilities = new PhoneNumberCapabilities()
+            .setCalling(PhoneNumberCapabilityType.INBOUND)
+            .setSms(PhoneNumberCapabilityType.INBOUND_OUTBOUND);
+        PhoneNumberSearchOptions searchOptions = new PhoneNumberSearchOptions().setAreaCode(areaCode).setQuantity(1);
```

Comment:
`.setAreaCode(areaCode).setQuantity(1);` is optional for toll free numbers. I think we should remove this and also line 18 so that we keep our samples as simple as possible.